frontend/src/pages/signin-page.tsx

```diff
@@ -73,7 +72,7 @@ function signin() {
     <div className="flex-grow cursor-default bg-white py-4 dark:bg-dark-card">
       <div className="m-4 mb-4 flex justify-center">
         <div className="flex w-full items-center justify-center">
-          <h2 className="w-3/4 pl-48 text-center text-lg font-bold text-black dark:text-dark-primary sm:text-xl">
+          <h2 className="text-center text-lg font-bold text-black dark:text-dark-primary sm:w-2/4 sm:pl-2 sm:text-xl md:w-3/4 md:pl-48">
             Sign in to WanderLust
```

Comment:
As discussed you would have to remove sm: because mobile screen would be less with than sm: and we need this for mobile screen right? which is default. so why do we need to add sm: correct?

Reply:
> As discussed you would have to remove sm: because mobile screen would be less with than sm: and we need this for mobile screen right? which is default. so why do we need to add sm: correct?

yup understood @krishnaacharyaa